uw-research-computing/hpc-transition-2023.md

```diff
@@ -0,0 +1,136 @@
+---
+highlighter: none
+layout: markdown-page
+title: Transitioning to a New HPC Cluster
+published: true
+---
+
+The Center for High Throughput Computing’s High Performance Cluster is being 
+replaced by a new High Performance System! All components of the system (execute 
+nodes, network, shared file system) are new and we expect an improved experience for our HPC users.
+
+ALL USERS OF THE EXISTING HPC SYSTEM WILL NEED TO MIGRATE TO THIS NEW CLUSTER. 
+Importantly, access to the existing cluster will be phased out in early 2022. 
+CHTC staff are here to assist with your transition.
```

Comment:
It would be nice to highlight this in some way but idk how the HTML gets constructed/deployed. @CannonLock can we do something like mkdocs admonitions (https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types)?